### PR TITLE
Judicial system/e2e tests

### PR DESCRIPTION
**Pull request by oddsson**
# Happy path e2e tests

https://app.asana.com/0/1199153462262248/1199662063468366

## What

Added happy path e2e tests

## Why

Better code

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -225,6 +225,7 @@ const CustomHeader = ({
         </Text>
       </div>
       <button
+        data-testid="datepickerIncreaseMonth"
```
Comment (barabrian) [inline libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx:228]: should this be more unique? and maybe be added to more critical elements inside the datepicker?
```suggestion
        data-testid={`${testId}-datepickerIncreaseMonth`}
```

**Review by barabrian — DISMISSED** [review]
LGTM. One comment regarding test id, maybe it's not crucial at this time but something to keep in mind

File: libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx (hunk 1 above)
Comment (oddsson) [reply libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx:228]: I think this only becomes an issue if you have two datepickers open at the same time and try to select this element. I think this is fine for now.

**Review by barabrian — DISMISSED** [review]

**Review by sindrig — APPROVED** [review]
Version bump LGTM

**Review by gudjong — APPROVED** [review]

**Review by barabrian — APPROVED** [review]

**Pull request merged**